install/Knative-with-Minishift.md

```diff
@@ -10,37 +10,64 @@ You can find [guides for other platforms here](README.md).
 
 ```shell
+
+# make sure you have  a profile is set correctly, e.g. knative
 minishift profile set knative
+
+# Pinning to the right needed OpenShift version, anything >= v3.11.0
+minishift config set openshift-version v3.11.0
+
+# minimum memory required for the minishift VM
 minishift config set memory 8GB
+
+# the minimum required vCpus for the minishift vm
 minishift config set cpus 4
+
+# extra disk size for the vm, default is 20g
+minishift config set disk-size 50g
+
+# caching the images that will be downloaded during app deployments
 minishift config set image-caching true
-minishift addon enable admin-user
-minishift addon enable anyuid
-```
 
-The above configuration ensures that Knative gets created in its own [minishift profile](https://docs.okd.io/latest/minishift/using/profiles.html) with 8GB of RAM and 4 vCpus. The image-caching helps in re-starting up the cluster faster every time.  The [addon](https://docs.okd.io/latest/minishift/using/addons.html) **admin-user** creates a default `admin` user with the role  cluster-admin and the  [addon](https://docs.okd.io/latest/minishift/using/addons.html) **anyuid** allows the `default` service account to run the application with uid `0`.
+# Add new user called admin with password admin having role cluster-admin
+minishift addons enable admin-user
 
-To start minishift:
+# Allow the containers to be run with uid 0
+minishift addons enable anyuid
 
-```shell
-minishift profile set knative
+# start minishift
 minishift start
 ```
 
-The command `minishift profile set knative` is required every time you start and stop minishift to make sure that you are on right `knative` minishift profile that was configured above.
+- The above configuration ensures that Knative gets created in its own [minishift profile](https://docs.okd.io/latest/minishift/using/profiles.html) called `knative` with 8GB of RAM, 4 vCpus and 50GB of hard disk. The image-caching helps in re-starting up the cluster faster every time.  
+- The [addon](https://docs.okd.io/latest/minishift/using/addons.html) **admin-user** creates a user called  `admin` with password  `admin` having the role of cluster-admin
+- The  [addon](https://docs.okd.io/latest/minishift/using/addons.html) **anyuid** allows the `default` service account to run the application with uid `0`
+
+> The command `minishift profile set knative` is required every time you start and stop minishift to make sure that you are on right `knative` minishift profile that was configured above.
 
 ## Configuring `oc` (openshift cli)
 
-Running the following command make sure that you have right version of `oc` and have configured the DOCKER daemon to be connected to minishift Docker.
+Running the following command make sure that you have right version of `oc` and have configured the DOCKER daemon to be connected to minishift docker.
 
 ```shell
 eval $(minishift docker-env) && eval $(minishift oc-env)
```

Comment:
This would not work for Windows users, or some non-bash shells. Commands `minishift docker-env` and `minishift oc-env` handles this internally, so better would be to ask users to execute `docker-env` and then execute its output again.